### PR TITLE
Added FACE_BBOX modality

### PR DESCRIPTION
**Pull request by Rauf-Kurbanov**
https://github.com/Synthesis-AI-Dev/face_api_dataset/issues/5

### CODE REVIEW

File: face_api_dataset/dataset.py [hunk 1]
```diff
@@ -606,4 +587,55 @@ def _open_modality(
                 info["facial_attributes"]["gaze"]["vertical_angle"],
             ]
             return np.array(gaze, dtype=np.float64)
+
+        if modality == Modality.FACE_BBOX:
+            segment_img, segment_mapping_int = self._read_segments(number, info)
+
+            face_segments = ["brow", "cheek_left", "cheek_right", "chin",
```
Comment (adavydow) [inline face_api_dataset/dataset.py:594]: I suggest making this a parameter (similar to segment map).

Comment (adavydow) [reply face_api_dataset/dataset.py:594]: Probably add padding as well

File: face_api_dataset/dataset.py [hunk 2]
```diff
@@ -140,6 +140,12 @@ class Modality(Enum):
     
     **Type**: `ndarray[float64]`. **Dimensions**: 2.
     """
+    FACE_BBOX = auto()
```
Comment (adavydow) [inline face_api_dataset/dataset.py:143]: We will need to regenerate the docs.

Comment (Rauf-Kurbanov) [reply face_api_dataset/dataset.py:143]: How can I do that?

File: face_api_dataset/dataset.py (hunk 1 above)
Comment (adavydow) [reply face_api_dataset/dataset.py]: I mean the option to increase the size of the bounding box.

File: face_api_dataset/dataset.py (hunk 2 above)
Comment (adavydow) [reply face_api_dataset/dataset.py:143]: I'll do this.

File: face_api_dataset/dataset.py [hunk 3]
```diff
@@ -140,6 +140,12 @@ class Modality(Enum):
...
+    """
+    Face bounding box in format x0 y0 x1 y1
```
Comment (adavydow) [inline face_api_dataset/dataset.py:145]: I assume its pixels? Is the format left-top-right-bottom?

File: face_api_dataset/dataset.py (hunk 2 above)
Comment (adavydow) [reply face_api_dataset/dataset.py:143]: `make html` in the docs directory will do the trick, but you'll need sphinx packages installed.

**Pull request merged**